src/boot/sysfail.h

```diff
@@ -0,0 +1,13 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "efivars-fundamental.h"
+
+typedef enum SysFailType {
+        SYSFAIL_NO_FAILURE = 0x0,
+        _SYSFAIL_MAX,
+        _SYSFAIL_INVALID = -EINVAL,
```

Comment:
EINVAL is a userspace thing, shouldn't be in src/boot/sysfail.h, i.e. a boot lodaer header